Sources/Plot/API/HTMLAttributes.swift

```diff
@@ -158,6 +158,11 @@ public extension Attribute where Context == HTML.LinkContext {
     static func color(_ color: String) -> Attribute {
         Attribute(name: "color", value: color)
     }
+
+    /// Assign a `crossorigin` attribute to the link.
+    static func crossorigin() -> Attribute {
```

Comment:
How about we also add a boolean argument to this function, to enable this attribute to be conditionally added?

For example, see the `controls` and `open` methods for reference, they both take a `Bool` argument and simply return `.empty` if the argument is `false`.